TypeCobol.LanguageServer/VsCodeProtocol/Text Document/TextDocumentIdentifier.cs

```diff
@@ -23,5 +23,17 @@ public TextDocumentIdentifier(string uri)
         {
             this.uri = uri;
         }
+
+        /// <summary>
+        /// Creates a new TextDocumentIdentifier literal.
+        /// @param uri The document's uri.
```

Comment:
Java syntax, we're speaking C# here 💥

Reply:
Changed here (but there are many @param in other classes)